CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## [2.1.0] - June 3, 2021:
+* SelectableHtml widget (supporting a subset of tags) which allow text selection
+* Fixed shrinkWrap to actually shrink the horizontal space
+* Support style tags to apply styling from inline css
+* Support applying styles from Flutter themes
+* Mouse hover on links when using Flutter Web
+* Allow custom anchor link tap implementations
+* Support additional list styling options
+* Fix several minor whitespace issues in text flow
+* Fixed specific colspan/rowspan usages in tables
+* Fixed whitespace issues around images
+* Swallow duplicate ids to prevent crashing the widget
+* Fixes crashing tap detection when using both link and image taps
+* Updates external dependencies
+* Raised minimum Flutter version to 2.2.0
+
```

Comment:
We also had stuff like fixing the crash when tapping on a linked image, support uppercase hex and hex with alpha, fixing br getting removed from the tree, fixing empty space when block elements were empty. Don't know if you want to include those, just thought I'd suggest it.

Reply:
@tneotia I wonder if there is some sort of automated tool to do this for us...... Y'know, closed issues, closed PRs, etc.

Reply:
Possibly, but we would probably have to go back in and change the data it pulled out to make it clear and concise.

I like to use the releases tab and hit the "x commits to master since this release" to get a quick list, but we didn't make a release for 2.0.0 on github.